src/Symfony/Component/DependencyInjection/Tests/Loader/FileLoaderTest.php

```diff
@@ -174,7 +177,8 @@ public function testRegisterClassesWithExcludeAsArray()
         $loader->registerClasses(
             new Definition(),
             'Symfony\Component\DependencyInjection\Tests\Fixtures\Prototype\\',
-            'Prototype/*', [
+            'Prototype/*',
+            [
```

Comment:
please revert

Reply:
Done :+1: Seems like I'm using a newer version of php-cs-fixer